develop/plone/functionality/controlpanel.rst

```diff
@@ -49,9 +49,7 @@ The add-on package in this case is called
 `silvuple <https://github.com/miohtama/silvuple>`_.
 
 In ``buildout.cfg``, make sure you have the ``extends`` line for
-Dexterity (see the
-`Dexterity installation guide
-<https://plone.org/products/dexterity/documentation/how-to/install>`_.
+Dexterity (see the :doc:`Dexterity installation guide </external/plone.app.dexterity/docs/install>`
```

Comment:
Close parenthesis. It might need a space before it, if it does not render correctly.

```suggestion
Dexterity (see the :doc:`Dexterity installation guide </external/plone.app.dexterity/docs/install>`)
```